### ISSUE TITLE
Build CarlaUE4 from source,make launch freezed.

### ISSUE THREAD

**Issue by jahad9819jjj** [carla-simulator/carla#1808]
Hi, I appreciate you to create Carla!
I try to build carla 0.9.x on Ubuntu16.0.4 by using UE4.22.

I follow the document and can run correctly.
but I did `make launch` and receive a below error.
Then launching CarlaUE4 freezed in the middle.

```
Error: CDO Constructor (CarlaMapGenerator): Failed to find StaticMesh'/Game/Carla/Static/RoadLines/St_Road_XCross_LaneMarking.St_Road_XCross_LaneMarking'
 Error: CDO Constructor (CarlaMapGenerator): Failed to find StaticMesh'/Game/Carla/Static/SideWalk/St_Road_XCross_Sidewalk4.St_Road_XCross_Sidewalk4'

```

**Comment by BenMSK** [comment]
I got similar errors but solved it. 
I think that the problem is caused by, mentioned files in 'Contents folder' don't exist.

The following commands for solving are, 
1) At the cloned folder, type "$make clean". It deletes all dependencies of UE4.22.
2) "$git pull" (if your version of Carla is not up-to-date.
3) "$./Update.sh" --> It will update all needed files into 'Contents folder'.
4) "make launch" again.

I am not sure that I solved it correctly, but it worked :)
